odoo/openupgrade/doc/source/modules100-110.rst

```diff
@@ -137,7 +137,7 @@ missing in the new release are marked with |del|.
 +-----------------------------------+-------------------------------------------------+
 |google_spreadsheet                 |                                                 |
 +-----------------------------------+-------------------------------------------------+
-|hr                                 |                                                 |
+|hr                                 | Done                                            |
```

Comment:
This should be `Nothing to do`

Reply:
Not if my change request is followed up ;-)

Reply:
I though it was a false positive. About `hr.module_install_notification`, it's harmless to let it, because it's not going to be sent anymore.

Reply:
True!